packages/web-components/fast-components/src/tree-item/tree-item.styles.ts

```diff
@@ -94,233 +102,199 @@ export const treeItemStyles: (
     css`
-    ${display("block")} :host {
-        contain: content;
-        position: relative;
-        outline: none;
-        color: ${neutralForegroundRest};
-        background: ${neutralFillStealthRest};
-        cursor: pointer;
-        font-family: ${bodyFont};
-        --expand-collapse-button-size: calc(${heightNumber} * 1px);
-        --tree-item-nested-width: 0;
-    }
+        ${display("block")} :host {
+            --expand-collapse-button-size: calc(${heightNumber} * 1px);
+            --tree-item-nested-width: 0;
+            contain: content;
+            position: relative;
+            outline: none;
+            color: ${neutralForegroundRest};
+            fill: currentcolor;
+            cursor: pointer;
+            font-family: ${bodyFont};
+        }
 
-    :host(:focus) > .positioning-region {
-        outline: none;
-    }
+        :host(:focus) > .positioning-region,
+        :host(:focus) .content-region {
+            outline: none;
+        }
 
-    :host(:focus) .content-region {
-        outline: none;
-    }
+        .positioning-region {
+            display: flex;
+            position: relative;
+            box-sizing: border-box;
+            background: ${neutralFillStealthRest};
+            border: calc(${strokeWidth} * 1px) solid transparent;
+            border-radius: calc(${controlCornerRadius} * 1px);
+            height: calc((${heightNumber} + 1) * 1px);
+        }
 
-    :host(:${focusVisible}) .positioning-region {
-        border: ${focusStrokeOuter} calc(${strokeWidth} * 1px) solid;
-        border-radius: calc(${controlCornerRadius} * 1px);
-        color: ${neutralForegroundRest};
-    }
+        :host(:not([disabled])) .positioning-region:hover {
+            background: ${neutralFillStealthHover};
+            color: ${neutralForegroundHover};
+        }
 
-    .positioning-region {
-        display: flex;
-        position: relative;
-        box-sizing: border-box;
-        border: transparent calc(${strokeWidth} * 1px) solid;
-        height: calc((${heightNumber} + 1) * 1px);
-    }
+        :host(:not([disabled])) .positioning-region:active {
+            background: ${neutralFillStealthActive};
+            color: ${neutralForegroundActive};
+        }
 
-    .positioning-region::before {
-        content: "";
-        display: block;
-        width: var(--tree-item-nested-width);
-        flex-shrink: 0;
-    }
+        :host(:${focusVisible}) .positioning-region {
+            background: ${neutralFillStealthFocus};
+            border-color: ${focusStrokeOuter};
+            box-shadow: 0 0 0 calc(${focusStrokeWidth} * 1px) ${focusStrokeOuter} inset;
+            color: ${neutralForegroundHover};
+        }
 
-    .positioning-region:hover {
-        background: ${neutralFillStealthHover};
-    }
+        :host(:not([disabled])[selected]) .positioning-region {
+            background: ${neutralFillStealthActive};
```

Comment:
In Fluent this is "secondary" fill. I don't think we have the same need for that in this FAST design, but it should be different and not `Active`. Let's use the regular `neutralFillRest` here.

Reply:
updated